scripts/g.download.location/g.download.location.html

```diff
@@ -11,6 +11,27 @@ <h2>DESCRIPTION</h2>
 The first directory which is a project is used.
 Other projects or any other files are ignored.
 
+<h2>EXAMPLES</h2>
+
+<h3>Download the full GRASS GIS sample database</h3>
+
+Download and unpack the GRASS GIS sample database into the user's
+HOME directory:
+
+<div class="code"><pre>
+g.download.location url=https://grass.osgeo.org/sampledata/north_carolina/nc_spm_full_v2alpha2.tar.gz path=$HOME
+</pre></div>
```

Comment:
Given the context and given that you do try to deal with that later anyway, it seems better to me to mention here that it is in an session.